util/topgen.py

```diff
@@ -1099,6 +981,145 @@ def main():
     # These modules are not templated, but are not in hw/ip
     generate_top_only(top_only_list, out_path, topname)
 
+    if pass_idx:
```

Comment:
ditto here. `pass_idx == 1` in this case. or we can pack two if s `pass_idx == 1 and args.top_ral`